website/docs/reference/commands/cmd-docs.md

```diff
@@ -26,7 +26,7 @@ dbt docs generate --no-compile
 ```
 
 ### dbt docs serve
-This command starts a webserver on port 8000 to serve your documentation locally. The webserver is rooted in your `target/` directory.
+This command starts a webserver on port 8000 to serve your documentation locally. Run `generate` before `serve` as `serve` depends on the catalog which `generate` produces. The webserver is rooted in your `target/` directory.
```

Comment:
```suggestion
This command starts a webserver on port 8000 to serve your documentation locally. The webserver is rooted in your `target/` directory. Be sure to run `dbt docs generate` before `dbt docs serve` because the  `generate` command produces a [catalog metadata artifact](/reference/artifacts/catalog-json) that the `serve` command depends upon. You will see an error message if the catalog is missing.  
```